client/src/resources/lang/en_us.json

```diff
@@ -147,7 +147,7 @@
     "role.bodyguard.dayTarget": "Special",
 
     "role.crusader.name": "Crusader",
-    "role.crusader.target": "Protect",
+    "role.crusader.target": "Guard",
     "role.crusader.dayTarget": "Special",
 
     "role.reveler.name": "Fisherman",
```

Comment:
change this to reveler idk why you made it fisherman

Reply:
I didn't, im pro reveler ill change